frontend/taipy-gui/src/components/Taipy/MenuCtl.tsx

```diff
@@ -46,6 +54,7 @@ const MenuCtl = (props: MenuCtlProps) => {
     useDispatchRequestUpdateOnFirstRender(dispatch, id, module, props.updateVars, props.updateVarName);
 
     const lovList = useLovListMemo(props.lov, defaultLov, true);
+    const lovSelectedIds = useLovListMemo(props.selectedIds, defaultSelectedIds, true);
```

Comment:
unnecessary

Reply:
value should be updated on the frontend by giving specifying rel_var so that LoVItems can be assigned to the value (multiple)